src/SignalR/clients/ts/FunctionalTests/func.jest.config.js

```diff
@@ -2,33 +2,20 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 module.exports = {
-    transformIgnorePatterns: [
+        transformIgnorePatterns: [
         // We reference the ESM output from tests and don't want to run them through jest as it won't understand the syntax
         ".*/node_modules/(?!@microsoft)"
     ],
     reporters: [
         "default",
-        ["../common/node_modules/jest-junit/index.js", { "outputDirectory": "../../../../../artifacts/log/", "outputName": `${process.platform}` + ".node.functional.junit.xml" }]
+        ["jest-junit", { "outputDirectory": "../../../../artifacts/log/", "outputName": `${process.platform}` + ".signalr.junit.xml" }]
```

Comment:
Why delete `..`? I don't think it's going to the correct dir anymore.

Reply:
Also this name conflicts with the name of the signalr unit tests. They explicitly had different names.

Reply:
I've reverted this bit.